### PR TITLE
feat: Allow user to define PHP executable path

### PR DESCRIPTION
**Pull request by zobo**

### CODE REVIEW

**Comment by zobo** [comment]
Created new PR due to accidentall closure of #61

File: src/extension.ts [hunk 1]
```diff
@@ -29,6 +30,14 @@ export function activate(context: vscode.ExtensionContext): void {
         config.path = context.asAbsolutePath(join('vendor', 'phpactor', 'phpactor', 'bin', 'phpactor'))
     }
 
+    if (!config.executablePath) {
+        const phpConfig = vscode.workspace.getConfiguration('php')
+        config.executablePath =
+            phpConfig.get<string>('executablePath') ||
+            phpConfig.get<string>('validate.executablePath') ||
```
Comment (dantleech) [inline src/extension.ts:37]: `validate.executablePath`?

File: package.json [hunk 2]
```diff
@@ -87,6 +87,14 @@
                     "default": null,
                     "description": "Specifies the path to the Phpactor binary"
                 },
+                "phpactor.executablePath": {
```
Comment (dantleech) [inline package.json:90]: maybe call this `phpBinaryPath`

**Review by dantleech — APPROVED** [review]
:+1: just a small CR

File: src/extension.ts (hunk 1 above)
Comment (zobo) [reply src/extension.ts:37]: php.executablePath and php.validate.executablePath are used as fallbacks and are "inherited" from Microsofts internal PHP tooling as well as vscode-php-debug and vscode-php-intellisense. https://code.visualstudio.com/docs/languages/php#_linting

File: package.json (hunk 2 above)
Comment (zobo) [reply package.json:90]: Suggested this to keep in line with Microsofts internal PHP linter naming...

I should definitely extend the readme with instructions tho.

**Comment by dantleech** [comment]
Thanks!

**Pull request merged**